execution/gethexec/blockchain.go

```diff
@@ -75,7 +75,7 @@ var DefaultCachingConfig = CachingConfig{
 	SnapshotRestoreGasLimit:            300_000_000_000,
 	MaxNumberOfBlocksToSkipStateSaving: 0,
 	MaxAmountOfGasToSkipStateSaving:    0,
-	StylusLRUCache:                     256,
+	StylusLRUCacheCapacity:             256,
```

Comment:
Suggestions on the default value?

Reply:
I am fine with 256 for now. If we frequently run out, we can change it in a later PR.